deepinv/models/flowunet.py

```diff
@@ -0,0 +1,523 @@
+# Unet copied from "A Variational Perspective on Diffusion-Based Generative Models and Score Matching", NeurIPS 2021, Huang et al
```

Comment:
@annegnx @SegoleneMartin THis should rather go in the docstring of the FlowUnet class IMO